pmd-java/src/test/kotlin/net/sourceforge/pmd/lang/java/ast/ASTTryStatementTest.kt

```diff
@@ -57,35 +65,45 @@ class ASTTryStatementTest : ParserTestSpec({
 
         "try (a){}" should matchStmt<ASTTryStatement> {
 
-            child<ASTResourceSpecification> {
-                child<ASTResources> {
-                    child<ASTResource> {
-                        child<ASTVariableReference> {}
-                    }
+            child<ASTResourceList> {
+                child<ASTResource> {
+                    it::isConciseResource shouldBe true
+                    it::getInitializer shouldBe variableRef("a")
```

Comment:
Can we test here `it::getStableName` as well? I'm not sure, I understand it.... 
In this case here, stable name will be simply "a", right?

In the following example: `try (a.b){}`, is the stable name then "a.b"?

Reply:
Yes it would be "a.b". The method just pretty prints the expression for concise resources. We should probably make a separate utility for that at some point in the future, that rules can reuse.